resources/metadata/sonarExecuteScan.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           - name: host
           - name: sonarServerUrl
         type: string
-        description: "The URL to the Sonar backend."
+        description: "The URL to the Sonar backend. The serverUrl parameter requires the [`instance`](#instance) parameter to be explicitly set to an empty string, as it will have no effect otherwise."
```

Comment:
isn't instance parameter deprecated? or its still valid for Jenkins?

Reply:
it is still valid and used in Jenkins